pkg-py/docs/context.qmd

```diff
@@ -2,13 +2,13 @@
 title: Provide context
 ---
 
-To improve the LLM's ability to accurately translate natural language queries into SQL, it often helps to provide relevant metadata. Querychat automatically provides things like column names and data types to the LLM, but you can enhance this further with additional context like [data descriptions](#data-description). You can also provide [custom instructions](#extra-instructions) to add additional behaviors and even supply a fully [custom prompt template](#custom-template), if desired.
+To improve the LLM's ability to accurately translate natural language queries into SQL, it often helps to provide relevant metadata. QueryChat automatically provides things like column names and data types to the LLM, but you can enhance this further with additional context like [data descriptions](#data-description). You can also provide [custom instructions](#extra-instructions) to add additional behaviors and even supply a fully [custom prompt template](#custom-template), if desired.
 
 All of this information is provided to the LLM as part of the **system prompt** -- a string of text containing instructions and context for the LLM to consider when responding to user queries.
 
 ## Default prompt
 
-For full visibility into the full system prompt that Querychat generates for the LLM, see the `system_prompt` property. This is useful for debugging and understanding exactly what context the LLM is using:
+For full visibility into the full system prompt that QueryChat generates for the LLM, see the `system_prompt` property. This is useful for debugging and understanding exactly what context the LLM is using:
```

Comment:
It feels weird to me that we're calling it `QueryChat` but the package name is `querychat`. `pip install QueryChat` works but `install.packages("QueryChat")` won't.

I like the effort for consistency! I also generally like the casing for accessibility reasons, but I'm not sure I really want to maintain separate formatting for the display name and package name, especially when it's `querychat` in both languages